modules/tinymce/src/themes/mobile/test/ts/module/test/theme/TestTheme.ts

```diff
@@ -3,15 +3,15 @@ import { Arr, Fun } from '@ephox/katamari';
 import { TinyApis, TinyLoader } from '@ephox/mcagar';
 
 import ThemeManager from 'tinymce/core/api/ThemeManager';
-import Features from 'tinymce/themes/mobile/features/Features';
-import FormatChangers from 'tinymce/themes/mobile/util/FormatChangers';
+import * as Features from 'tinymce/themes/mobile/features/Features';
+import * as FormatChangers from 'tinymce/themes/mobile/util/FormatChangers';
 
 interface Realm {
   system: () => Gui.GuiSystem;
   socket: () => AlloyComponent;
 }
 
-const name = 'test';
+const sName = 'test';
```

Comment:
One more variable name that we possibly should rethink?

Reply:
done